src/slash/drop.ts

```diff
@@ -32,7 +46,37 @@ export class DropCommand extends AutocompletableCommand {
 
 	protected override async execute(interaction: ChatInputCommandInteraction<"cached">): Promise<void> {
 		await interaction.deferReply();
-		const tournamentName = interaction.options.getString("tournament", true);
+		const tournamentName = interaction.options.getString("tournament");
+		if (!tournamentName) {
+			const players = await ManualParticipant.find({
+				where: { discordId: interaction.user.id },
+				relations: ["tournament"]
+			});
+
+			if (players.length < 1) {
+				await interaction.editReply(`You are not in any tournaments.`);
+				return;
+			}
+
+			if (players.length > 1) {
+				if (!interaction.guild.commands.cache.size) {
+					await interaction.guild.commands.fetch();
+				}
+				const id = interaction.guild.commands.cache.find(command => command.name === "drop")?.id || "";
+				const mention = chatInputApplicationCommandMention("forcedrop", id);
+				await interaction.editReply(
+					`You are in multiple tournaments. Use the \`tournament\` option in ${mention} to specify.`
+				);
```

Comment:
I don't think we need this, because the user is already using the /drop command, and also participants shouldn't be able to use /forcedrop.